app/src/renderer/components/staking/LiValidator.vue

```diff
@@ -1,28 +1,18 @@
 <template lang='pug'>
-li.li-validator(:class='styles')
-  router-link(:to="{ name: 'validator', params: { validator: validator.id }}")
-    .li-validator__value.name
+tr.li-validator
+  td.li-validator__moniker-container
+    img.li-validator__avatar(v-if="validator.keybase" :src="validator.keybase.avatarUrl" width="48" height="48")
+    img.li-validator__avatar.no-img(v-else src="~assets/images/validator-icon.svg" width="48" height="48")
+    .li-validator__name-container
       span.validator-profile__status(v-bind:class="statusColor" v-tooltip.top="status")
-      img.avatar(v-if="validator.keybase" :src="validator.keybase.avatarUrl" width="48" height="48")
-      img.avatar(v-else src="~assets/images/validator-icon.svg" width="48" height="48")
-      .vert
-        .top {{ validator.description.moniker }}
-        short-bech32(:address="validator.operator_address")
-    .li-validator__value.your-votes
-      span {{ yourVotes.isLessThan(0.01) && yourVotes.isGreaterThan(0) ? '< ' + num.shortNumber(0.01) : num.shortNumber(yourVotes) }}
-    .li-validator__value.your-rewards
-      span n/a
-    .li-validator__break: span
-    .li-validator__value.percent_of_vote
-      span {{ validator.percent_of_vote ? validator.percent_of_vote : `n/a` }}
-    .li-validator__value.uptime
-      // add .green .yellow or .red class to this span to trigger inidication by color
-      span {{ uptime }}
-    .li-validator__value.commission
-      span {{ commission }}
-    .li-validator__value.slashes
-      // add .green .yellow or .red class to this span to trigger inidication by color
-      span {{ slashes }}
```

Comment:
too many `<span>`'s

Reply:
You having a conversation with yourself, mister?